tests/common/mod.rs

```diff
@@ -132,13 +132,36 @@ impl TestDir {
         let mut cmd = Self::init_cmd();
         let canonical_current_dir = canonicalize(&self.current_dir).expect("canonical current dir");
         cmd.current_dir(&canonical_current_dir)
-            .args(&["-f"])
+            .args(&["-f", "--no-backup"])
             .assert()
             .failure()
             .code(1)
             .stdout(expected_output);
     }
 
+    /// Run the default CLI binary, with "-f", in this TestDir and checks a backup file is created
+    pub fn test_command_fix_backs_up_files(self) {
```

Comment:
I think creating a function to run in only one test is a bit overkill 🤔

Reply:
i think you're right, but see my comment below about the reasons why I made one (also, I updated the function based on your comment below)